components/configurations-generator/Jenkinsfile

```diff
@@ -54,19 +54,19 @@ podTemplate(label: label,
                         }
 
                         stage("install dependencies $application") {
-                            execute("make resolve", workDir, componentDir)
+                            execute("dep ensure --vendor-only -v", workDir, componentDir)
```

Comment:
consider combining stages and using before-commit.sh

Reply:
From what I have been told, we want to implement the least amount of changes, and this won't remove existing stages

Reply:
I agree with @Demonsthere. Jenkinsfiles are almost extinct, so I'd say every approach that just keep them running is a valid one.